docs/background/disaster-recovery.md

```diff
@@ -0,0 +1,85 @@
+# Disaster recovery
+
+When you [create a new server](../../howto/openstack/nova/new-server) in
+{{brand}}, you will notice an option named **Disaster recovery**. That
+particular option is enabled by default.
+
+![Disaster recovery is enabled by default for new
+servers](assets/disaster-recovery-option-on-by-default.png)
+
+Even if you choose to disable it for a particular server, keep in mind
+that you have the option to enable it at a later time.
+
+![The disaster recovery feature can be manually activated for any
+server](assets/disaster-recovery-option-manual-activation.png)
+
+In the following, we explain what this option does, how it works in the
+background, and why you should consider enabling it.
+
+## What it is
+
+The *Disaster Recovery* (DR) feature, is available via the {{gui}} and
+applies to servers and volumes that use our [Ceph](https://ceph.io)
+backend. That would be **all** servers but the ones of the `s`
+[flavor](../../reference/flavors/#compute-tiers).
+
+## How it works
+
+As soon as you enable DR for a server or a single volume, then you start
```

Comment:
"As soon as" followed by "then" sounds a little awkward to me.

```suggestion
As soon as you enable DR for a server or a single volume, you start
```

Reply:
It is indeed :$